### ISSUE TITLE
Remove deprecated t.w.static.FileTransfer

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5651]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#5651|
|Type|task|
|Created|2012-04-22 15:08:32Z|
|Branch|https://github.com/twisted/twisted/tree/remove-filetransfer-5651|

`FileTransfer` in `twisted.web.static` was deprecated in Twisted 9.0 ([#3897](https://github.com/twisted/twisted/issues/3897)/r27029) and can be removed.

<details><summary>Searchable metadata</summary>

```
trac-id__5651 5651
type__task task
reporter__thijs thijs
priority__low low
milestone__ 
branch__branches_remove_filetransfer_5651 branches/remove-filetransfer-5651
branch_author__thijs thijs
status__closed closed
resolution__fixed fixed
component__web web
keywords__easy easy
time__1335107312000000 1335107312000000
changetime__1338135699000000 1338135699000000
version__None None
owner__thijs thijs
cc__jknight
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

(In [34313]) Branching to 'remove-filetransfer-5651'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [34314]) remove FileTransfer, add news file. refs [#5651](https://github.com/twisted/twisted/issues/5651)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

Up for review.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set owner to @thijstriemstra|
|-|-|

Looks great, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_6"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [34462]) Merge remove-filetransfer-5651: Remove twisted.web.static.FileTransfer, deprecated since 9.0. Use a subclass of StaticProducer instead.

Author: thijs
Reviewer: therve
Fixes: [#5651](https://github.com/twisted/twisted/issues/5651)
